pkgs/development/ocaml-modules/ounit2/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, buildDunePackage, fetchurl, stdlib-shims }:
 
 buildDunePackage rec {
-  minimumOCamlVersion = "4.02.3";
+  minimumOCamlVersion = "4.04";
```

Comment:
This change “removes” many packages: nixpkgs-review lists 92 of them. Is it an issue?

Reply:
`nixpkgs-review` interestingly didn't list any for me, did you use any special options or do `rev` and `pr` behave differently?

If you think, it's worth it to fix 4.02 and 4.04 OCaml packages that depend on ounit, I'll go through them and add a

```nix
  doCheck = lib.versionAtLeast ocaml.version ounit.minimumOCamlVersion;
```

I myself really am not sure, it seems kinda sensible, but I guess if we do this for every such increase of `minimumOCamlVersion` this becomes a huge task?!